### DISCUSSION TITLE
RportCLI tool -  https proxy tunnels (reverse proxy function)

### DISCUSSION THREAD

**Discussion by msbusk** [cloudradar-monitoring/rportcli#11] [Ideas]
It would be great if it was possible to create https proxy tunnels(reverse proxy function) by using rportcli.
We often use rportcli to make tunnels .

**Comment by thorstenkramm** [comment]
This has been implemented in [0.4.0](https://github.com/cloudradar-monitoring/rportcli/releases/tag/0.4.0)

**Reply by msbusk** [reply]
It's perfect -  thank you. 

We have implemented the new version and it works perfectly.
